### PR TITLE
align with latest console-ui

### PR DESCRIPTION
**Pull request by ruromero**
Sync with latest changes from console project

Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

### CODE REVIEW

File: deploy/ui/form.json [hunk 1]
```diff
@@ -120,7 +141,6 @@
                 {
                   "label": "url",
                   "type": "text",
-                  "required": true,
```
Comment (bmozaffa) [inline deploy/ui/form.json]: Why are these 3 set to not being required any longer?

File: deploy/ui/form.json [hunk 2]
```diff
@@ -19,33 +19,56 @@
           "originalJsonPath": "$.spec.environment",
           "type": "dropDown"
         },
+
         {
-          "label": "Image Registry",
-          "default": "rhpam-trial",
+          "label": "External registry",
```
Comment (bmozaffa) [inline deploy/ui/form.json]: Is there a technical restriction to using the same label of "Image Registry"?

Comment (ruromero) [reply deploy/ui/form.json]: No, there isn't

File: deploy/ui/form.json (hunk 1 above)
Comment (ruromero) [reply deploy/ui/form.json]: Restored

**Pull request merged**